src/Core/Import/EntityField/Provider/CombinationFieldsProvider.php

```diff
@@ -54,48 +54,48 @@ public function __construct(TranslatorInterface $translator)
     public function getCollection()
     {
         $fields = [
-            new EntityField('id_product', $this->trans('Product ID')),
-            new EntityField('product_reference', $this->trans('Product Reference')),
-            new EntityField('group', $this->trans('Attribute (Name:Type:Position)'), '', true),
-            new EntityField('attribute', $this->trans('Value (Value:Position)'), '', true),
-            new EntityField('supplier_reference', $this->trans('Supplier reference')),
+            new EntityField('id_product', $this->trans('Product ID', 'Admin.Advparameters.Feature')),
```

Comment:
`Admin.Advparameters.Feature` is already the default domain in this class, although something linked to `Product` would seem more logical

Reply:
> I didn't comment them all but a few calls have too many parameters (domain already present)

Yes, that's a mistake indeed. I'm going to fix t.

> And I'm wondering about the use of `Admin.Advparameters.Feature` averywhere? Is this because it's for the Import feature? Maybe some of these keys already exist in other domain nonetheless

I did it because this is the domain being used, as you can see from https://github.com/PrestaShop/PrestaShop/blob/7ab0cae901f6b932414415fa173c5802818f6ee6/src/Core/Import/EntityField/Provider/CategoryFieldsProvider.php#L90
```
/**
     * A shorter name method for translations.
     *
     * @param string $id translation ID
     * @param string $domain translation domain
     *
     * @return string
     */
    private function trans($id, $domain = 'Admin.Advparameters.Feature')
    {
        return $this->translator->trans($id, [], $domain);
    }
```

This private `trans()` saves a few developer time but it cannot be parsed by the Translations Extraction tool. That's why I added the domains manually.

Reply:
Oh I see that's why you had to add the domains everywhere. Then you should probably remove the default value in the `trans` methods to avoid misunderstanding and missing domains.
That said, I know it was used up until now but I'm still wondering if `Admin.Advparameters.Feature` is the appropriate domain in all those cases, we should wait for @LouiseBonnard approval.